cx_Freeze/freezer.py

```diff
@@ -215,7 +227,7 @@ def _GetDefaultBinExcludes(self):
         if sys.platform == "win32":
             return ["comctl32.dll", "oci.dll", "cx_Logging.pyd"]
         else:
-            return ["libclntsh.so", "libwtc9.so"]
+            return ["libclntsh.so", "libwtc9.so", "ldd"]
```

Comment:
What is the meaning of the addition of "ldd"? What would request its inclusion anyway?

Reply:
On Alpine Linux ldd is a reference to a system library (virtual library maybe)

Reply:
See #513 for more reference

Reply:
Previously I had a patch #517 just for that, but I decided to put it together.